modules/account-map/modules/iam-roles/variables.tf

```diff
@@ -21,3 +15,24 @@ variable "root_account_stage_name" {
   description = "The stage name for the root account"
   default     = "root"
 }
+
+variable "root_account_tenant_name" {
+  type        = string
+  description = <<-EOT
+  The tenant name for the root account.
+
+  If the `tenant` label is not used, set this to `null`.
+  EOT
+  default     = "mgmt"
+}
```

Comment:
I would prefer to have and use just the one `global_tenant_name`. So far that has been sufficient.